### PR TITLE
docs: cluster-level resource propagation pause and resume capabilities

### PR DESCRIPTION
**Pull request by a7i**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:

Document work dispatching resume and pause capabilities

**Which issue(s) this PR fixes**:
Part of https://github.com/karmada-io/karmada/issues/5217

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/userguide/scheduling/resource-propagating.md [hunk 1]
```diff
@@ -671,3 +671,74 @@ spec:
...
+spec:
+ #...
+ placement:
+  clusterAffinity:
+   exclude:
+    - member1
+    - member3
+  #...
+  suspension:
+    clusterNames: ["member3"]
```
Comment (RainbowMango) [inline docs/userguide/scheduling/resource-propagating.md:720]:
```suggestion
    dispatchingOnClusters: ["member3"]
```
sync with the latest field name.

**Review by RainbowMango — COMMENTED** [review]
Just a nit, otherwise LGTM.
Great thanks.

File: docs/userguide/scheduling/resource-propagating.md [hunk 2]
```diff
@@ -671,3 +671,74 @@ spec:
...
 The `nginx` deployment in `default` namespace will be propagated to cluster `member2`.
 
+
+## Suspend and resume of resource propagation
+
+The `.spec.suspension` field in the `PropagationPolicy` and `ClusterPropagationPolicy` allows suspending and resuming resource propagation to one or more clusters.
+
+### Suspend resource propagation to all clusters
+
+To suspend resource propagfation work to all member clusters, use the following configuration:
```
Comment (XiShanYongYe-Chang) [inline docs/userguide/scheduling/resource-propagating.md:681]: propagfation -> propagation

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks a lot
/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**